### DISCUSSION TITLE
用vue3最新框架，选择ts，vite打包的时候，源码里面报错

### DISCUSSION THREAD

**Discussion by Wd13869167203** [zouyaoji/vue-cesium#374] [General]
这是报错信息
![image](https://user-images.githubusercontent.com/31448471/229763306-73c31f1f-18d1-4300-a4d6-bf322b4493e3.png)

**Comment by zouyaoji** [comment]
建议你参考一下

https://github.com/zouyaoji/vue-cesium-demo

如果仍有问题，请提供复现代码

**Reply by MangMax** [reply]
我一直在开发的项目也突然出现了问题
![image](https://user-images.githubusercontent.com/68525017/233899209-957c5f00-f4f1-47c1-aced-792327c76ec2.png)
疑似3.1.7升级之后出现的，之前ts提示都是正常的，现在一直找不到，但不影响功能

**Reply by houluy** [reply]
我目前也遇到相同的问题，只是ts部分有问题。程序能够正常运行。
仓库的架构貌似修改了，以前版本VcReadyObject需要`import type { VcReadyObject } from 'vue-cesium/es/utils/types'`，types是`types.d.ts`文件。现在这个文件变成了.map？不清楚该怎么使用。

**Reply by zouyaoji** [reply]
嗯 是个bug，我今晚修复一下
